Makefile

```diff
@@ -14,5 +14,9 @@ parsers/tree-sitter-typescript.wasm: node_modules/tree-sitter-typescript/typescr
 	mv $(notdir $@) $@
 
 parsers/tree-sitter-tsx.wasm: node_modules/tree-sitter-typescript/tsx/package.json
+	npx tree-sitter build-wasm $(dir $^)
+	mv $(notdir $@) $@
+
+parsers/tree-sitter-c_sharp.wasm: node_modules/tree-sitter-c-sharp/package.json
 	npx tree-sitter build-wasm $(dir $^)
 	mv $(notdir $@) $@
```

Comment:
What if you did

```
mv tree-sitter-c_sharp.wasm $@
```

And then used `c-sharp` for the language name at top and for target? Then you can avoid that odd snake / kebab mixture. Does that make sense?

Reply:
it seemed a bit weird for me to use `parsers/tree-sitter-c-sharp.wasm` as the target name, since there's actually no file called that, but I see what you mean with the $@. I'll give it a shot